scripts/remove-dashboard.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+kubectl --namespace kube-system delete deployment kubernetes-dashboard
+kubectl --namespace  kube-system delete svc kubernetes-dashboard
```

Comment:
this can be commented out or disable in the pipeline (i.e. it doesn't work)

Reply:
i'll look to update the pipeline for this to disable this feature